reflex-algos/src/main/python/mlops/parallelm/mlops/events/event_broker.py

```diff
@@ -80,7 +80,7 @@ def _add_node_and_agent_cols(self, df):
                 node = ion.node_by_pipe_inst_id[pipe_inst]
                 nodes_list.append(node.name)
             else:
-                raise MLOpsException("Error pipeline instance id is not found {}".format(pipe_inst))
+                nodes_list.append('N/A')
```

Comment:
But what does exactly do? - Addition of such node?

Reply:
i believe the intent was to add node info into the dataframe - but there are several events without a pipeline instance-id (start event, stop event etc). so in these cases, we append 'n/a'